norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,318 @@
+            <text prefix=" Årg. " variable="volume"/>
+            <text prefix=" " macro="issued"/>
+            <text prefix=", s. " variable="page"/>
+            <choose>
+              <if variable="DOI" match="none">
+                <text prefix=", " variable="URL"/>
+              </if>
+              <else>
+                <text prefix=", " variable="DOI"/>
+              </else>
+            </choose>
+            <text prefix=" " macro="retrieved-from"/>
+          </group>
+        </if>
+        <else-if type="article-newspaper article-magazine" match="any">
+          <group suffix=".">
+            <text macro="author-full"/>
+            <text prefix=". " macro="issued"/>
+            <text prefix=" " macro="title"/>
+            <text prefix=", " font-style="italic" variable="container-title"/>
+            <choose>
+              <if variable="URL" match="none">
+                <text prefix=", " macro="issued-full-date"/>
+              </if>
+              <else>
+                <text prefix=", " variable="URL"/>
+                <text macro="accessed-date"/>
+              </else>
+            </choose>
+          </group>
+        </else-if>
+        <else-if type="book thesis" match="any">
+          <group suffix=".">
+            <text macro="author-full"/>
+            <text prefix=". " macro="title"/>
+            <text prefix=", " suffix=" utg." variable="edition"/>
+            <text prefix=", " suffix=":" variable="publisher-place"/>
+            <text prefix=" " variable="publisher"/>
+            <text prefix=", " macro="issued-no-parenthesis"/>
+          </group>
+        </else-if>
+        <else-if type="chapter paper-conference" match="any">
+          <group suffix=".">
+            <text macro="author-full"/>
+            <text prefix=". " macro="title"/>
+            <text prefix=", i " font-style="italic" variable="container-title"/>
+            <text prefix=", " suffix=" (red.)" macro="editor"/>
+            <text prefix=", " suffix=". utg.," variable="edition"/>
+            <text prefix=", " suffix=": " variable="publisher-place"/>
+            <text variable="publisher"/>
+            <text prefix=", " macro="issued-no-parenthesis"/>
+            <text prefix=", " variable="URL"/>
+            <text prefix=" " macro="accessed-date"/>
+          </group>
+        </else-if>
+        <else-if type="legislation bill" match="any">
+          <group display="left-margin">
+            <text macro="issued-no-parenthesis"/>
+          </group>
+          <group display="right-inline" suffix=".">
+            <text macro="title"/>
+          </group>
+        </else-if>
+        <else-if type="legal_case">
+          <text macro="title"/>
+        </else-if>
+        <else-if type="personal_communication">
+          <text macro="author-full"/>
+          <text macro="accessed-date"/>
+        </else-if>
+        <else>
+          <group display="left-margin">
```

Comment:
This seems odd, e.g. for webpages? What's the logic here?

Reply:
I honestly do not understand this part of the code. The CSL spec isn't very clear on what display="left-margin does". Could you give me helping hand in understanding?

Reply:
Left-margin/right-inline is, basically:
left margin -tab- right-inline
So, e.g., what you'd see in a numeric bibliography where the 1. is in the margin and the rest of the bibliography entry is one tab off and itself left-aligned. I don't understand what it's doing here. Especially not for the webpages, but also not really for the legal sources. Unless the styleguide requires some weird formatting/tabs/indenting, I'd remove all display elements.

Reply:
Removed the seemingly unnecessary display elements now. Feel free to mark as resolved if it looks OK.